LDAR_Sim/src/methods/crew.py

```diff
@@ -97,8 +105,9 @@ def work_a_day(self, site_pool, candidate_flags=None):
             #     self.worked_today = True
 
         if self.worked_today:
-            cur_timestep = self.state['t'].current_timestep
             self.schedule.end_day(site_pool, itinerary)
+            # record crews' last coordinates of the day 
+            self.timeseries['Crew_footprint'][cur_timestep].append((self.lon,self.lat,self.id,self.crew_type))
```

Comment:
consider moving into self.schedule.end_day because it is mobile specific

Reply:
We can't move it to self.schedule.end_day since the timeseries is not in the mobile crew.

Reply:
Crew's lats and lons need to be saved under the timeseries['crew_footprint']